testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -102,6 +102,7 @@
     "PRM12", "TRJ12_bz2",  # Amber (v12 format, Issue 100)
     "PRMncdf", "TRJncdf", "NCDF",  # Amber (netcdf)
     "PFncdf_Top", "PFncdf_Trj", # Amber ncdf with Positions and Forces
+    "CPPTRAJ_TRAJ_TOP", "CPPTRAJ_TRAJ_1", "CPPTRAJ_TRAJ_2", # Amber ncdf extracted from CPPTRAJ
```

Comment:
If I may ask, what is the difference between traj 1 and 2? Could you not just chain traj 1 twice?

Reply:
You are way smarter than me, you are absolutely right :-)

Reply:
Definitely not smarter 🤣 I just spend far too much time on this repository.